presto-product-tests/README.md

```diff
@@ -0,0 +1,37 @@
+# Presto product tests
+
+## Test configuration
+
+Test assume that you have hadoop and presto cluster running. For setting up development environment we
```

Comment:
> Could you describe your motivation for why you dislike launching product tests from Maven. Why is launching integration tests OK and product tests not? 

The main difference is that unit tests and integration tests in Presto test the *current* codebase, whereas the product tests in this suite are meant to test an external presto instance. So, while it's possible to run unit and integration tests as part of the build/release, it's not possible to do the same with product tests.

The way I see it, the main benefit from launching tests from the IDE is that you can step though the code to debug issues. This works well if it's unit or in-process integration tests. For tests that drive an external system, you'd only be debugging the testing framework or queries, no the target system. 

For us, requiring a source checkout to run the test suite is a big limitation. It's easier to package a tool and deploy it to our testing infrastructure than to push a copy of the code and tools necessary to be able to run a full maven build. It's also harder to know what version of the test suite is being exercised, etc.

> Currently we can launch product tests using both IDE (e.g IntelliJ) and Maven. Launching product tests from IntellJ is extremely useful when testing Presto in local development environment. But some people prefer to start tests from command line, therefore we would prefer to keep Maven way of launching product tests too.

Agreed, although this is orthogonal to whether it uses TestNG+surefire or a custom "application" that drives the test (like we do with the verifier).